go/service/kbfs_mount.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package service
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/libkb"
+	//	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+)
+
+type KBFSMountHandler struct {
+	*BaseHandler
+	libkb.Contextified
+}
+
+func NewKBFSMountHandler(xp rpc.Transporter, g *libkb.GlobalContext) *KBFSMountHandler {
+	return &KBFSMountHandler{
+		BaseHandler:  NewBaseHandler(xp),
+		Contextified: libkb.NewContextified(g),
+	}
+}
+
+func (h *KBFSMountHandler) GetCurrentDriveLetter(ctx context.Context) (res string, err error) {
```

Comment:
So, the service would return nothing for "current", in which case kbfs would pick one and save it as the current. OK.
It sounds like @taruti wants a non-mounting mode? I think it is better to have a separate command for that, since we need empty mount to mean that no default has been established yet.